src/main.cpp

```diff
@@ -0,0 +1,82 @@
+#include <SPI.h>        
+#include <NativeEthernet.h>
+#include <NativeEthernetUdp.h>
+
+constexpr int SUCCESSFUL = 1;
+constexpr int FAILIURE = 0;
```

Comment:
```suggestion
constexpr int FAILURE = 0;
```

Reply:
If one is going to set states, it is better to use enum.

enum result = { FAILURE, SUCCESS, INTERRUPTED, ABORTED... }

If one is just setting a flag for success/failure, it would be better to use a boolean
bool result = random_function(...)  
instead of 0 or 1. Remember that this is cpp and not c